gatling/user-files/DockerFile/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "dockerfile",
```

Comment:
Shouldn't you use the `package.json` file at the root of this project instead?
(https://github.com/kuzzleio/kuzzle-benchmark/blob/gatling-scenario/package.json)

Also, this PR now uses `request-promise` but that module still isn't listed in that package.json file.

Reply:
Mistake from me : i forgot to commit the right package.json.